### PR TITLE
API usage example

### PR DESCRIPTION
**Pull request by simleo**
User-oriented API usage example based on the [IWC workflow Travis build](https://travis-ci.org/github/crs4/iwc) from the [travis branch in the crs4 fork](https://github.com/crs4/iwc/tree/travis). The workflow is registered at https://dev.workflowhub.eu/workflows/157 and pre-submitted by me to the [dev instance of LifeMonitor](https://api.dev.lifemonitor.eu/).

The example needs to be checked by someone with a different LifeMonitor / WorkflowHub identity than me.

### CODE REVIEW

**Comment by simleo** [comment]
Re: the `last_logs` deletions in the notebook, see #94 -- the example becomes very hard to follow visually if those fields are not removed.

File: examples/LifeMonitorAPI.ipynb [hunk 1]
```diff
@@ -0,0 +1,727 @@
...
+    "for b in latest_builds['items']:\n",
+    "    del b['last_logs']\n",
+    "latest_builds['items'][:3]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pick a build a get the details."
```
Comment (kikkomep) [inline examples/LifeMonitorAPI.ipynb:596]: "a get" ---> "and get"

Comment (simleo) [reply examples/LifeMonitorAPI.ipynb]: Fixed in cb7090242725ef70b51c52e8c7decf80432836a6.

**Comment by simleo** [comment]
Update: changed the call to get the full log. Had to set a "large" `limit_bytes` of 4 MB (I think that's the max for Travis logs) since there is currently no way to get the full log (see #97).

**Comment by ilveroluca** [comment]
* How about adding the workflow registration step?
* Small comment:  it might be more intuitive for a user to pick a registry by  `if _["name"] ==  "wfhubdev"` rather than by `_["type"] == "seek"`.  Many may not even know what Seek is.

**Comment by ilveroluca** [comment]
In addition to saying "Log in to Life Monitor", add another point saying "connect your account to the WorkflowHub registry"

**Comment by simleo** [comment]
> it might be more intuitive for a user to pick a registry by  `if _["name"] ==  "wfhubdev"` rather than by `_["type"] == "seek"`.  Many may not even know what Seek is.

Unfortunately the name could be anything. I changed the snippet to explicitly pick the wfhub dev instance by its URL.

**Review by ilveroluca — APPROVED** [review]
